components/molecules/BenefitCardHeaderSummary.js

```diff
@@ -1,15 +1,42 @@
 import { Link } from '@dts-stn/decd-design-system'
 import propTypes from 'prop-types'
 import { SummaryTypes } from '../../objects/UniversalBenefit'
+import en from '../../locales/en'
+import fr from '../../locales/fr'
+import { formatDate } from '../organisms/DashboardUtils'
 
 export default function BenefitCardHeaderSummary(props) {
   const t = props.locale
   const typesWithLinks = [SummaryTypes.PaymentAmount, SummaryTypes.LatestStatus]
 
+  const getBenefitCardValue = () => {
+    switch (props.summary.type) {
+      case SummaryTypes.PaymentAmount:
+        return t.netAmount.replace('{0}', props.summary.value)
+        break
+      case SummaryTypes.NextPayment:
+      case SummaryTypes.NextReport:
+      case SummaryTypes.LatestStatus:
+        return formatDate(props.summary.value)
+        break
+      default:
+        return null
+        break
+    }
+  }
+
   return (
     <div>
       <p>{t[props.summary.type].title}</p>
-      <p className="font-bold">{props.summary.value}</p>
+      <p
+        className={
+          props.summary.type === SummaryTypes.PaymentAmount
```

Comment:
Would it be possible to re-use the same case/switch to define both the value and the style class at the same time?